src/jit/lsraarm.cpp

```diff
@@ -727,6 +727,31 @@ void Lowering::TreeNodeInfoInit(GenTree* tree)
             info->internalIntCount = 1;
             break;
 
+        case GT_COPY:
+            info->srcCount = 1;
+            info->dstCount = 1;
+#ifdef ARM_SOFTFP
+            if (tree->TypeGet() == TYP_LONG) // Actually was `double` type
+            {
+                info->dstCount = 2;
+            }
+#endif
+            break;
+
+        case GT_PUTARG_REG:
+#ifdef ARM_SOFTFP
+            if (tree->TypeGet() == TYP_LONG) // Actually was `double` type
```

Comment:
(e.g. "This case currently only occurs for double types that are passed as TYP_LONG; actual long types would have been decomposed by now."